ysh/expr_eval.py

```diff
@@ -1260,6 +1260,10 @@ def _EvalExpr(self, node):
                 i2 = _ConvertToInt(self._EvalExpr(node.upper),
                                    'Range end should be Int', node.op)
 
+                # i2 is a big int so we don't have to worry about overflow
```

Comment:
yeah it should be a BigInt, but isn't yet.

The first step may be to raise a hard error for overflow, which is OK

Also unfortunately the ranges are still truncated in the lines below, I think we should fix that, though not sure if it should come before or after the BigInt changes mentioned

Reply:
Okay, I've removed the comment and left the TODO below it.

Where are we at with a BigInt implementation? I remember there were a few Zulip threads a while back.